src/main/java/frc/robot/subsystems/Drive.java

```diff
@@ -303,18 +304,27 @@ public void end(final boolean interrupted) {
     }
 
     public CommandBase autonomousCommand(String trajectoryName) {
-        return autonomousCommand(trajectoryName, true);
+        return autonomousCommand(trajectoryName, true, true);
     }
 
-    public CommandBase autonomousCommand(String trajectoryName, Boolean resetPose) {
-
-        String trajectoryJSON = "paths/" + trajectoryName + ".wpilib.json";
+    public CommandBase autonomousCommand(String trajectoryName, Boolean resetPose, boolean isJson) {
         Trajectory autoTrajectory = new Trajectory();
-        try {
-            Path trajectoryPath = Filesystem.getDeployDirectory().toPath().resolve(trajectoryJSON);
-            autoTrajectory = TrajectoryUtil.fromPathweaverJson(trajectoryPath);
-        } catch (IOException ex) {
-            DriverStation.reportError("Unable to open trajectory: " + trajectoryJSON, ex.getStackTrace());
+        if (isJson) {
```

Comment:
Do we plan on sticking with just JSON or just CSV, or supporting both long term?